src/screens/Ticketing/Purchase/Confirmation/index.tsx

```diff
@@ -87,14 +91,33 @@ const Confirmation: React.FC<ConfirmationProps> = ({
   );
 
   const vatAmount = totalPrice * (vatPercent / 100);
-  const vatAmountString = vatAmount.toLocaleString(language, {
-    minimumFractionDigits: 2,
-    maximumFractionDigits: 2,
-  });
-  const vatPercentString = vatPercent.toLocaleString(language, {
-    minimumFractionDigits: 0,
-    maximumFractionDigits: 2,
-  });
+
+  /*
+  Todo: toLocaleString not working on Android until React Native 0.65. Remove manual
+    handling when RN 0.65 is used.
+  */
+  // const vatAmountString = vatAmount.toLocaleString(language, {
+  //   minimumFractionDigits: 2,
+  //   maximumFractionDigits: 2,
+  // });
+  // const vatPercentString = vatPercent.toLocaleString(language, {
+  //   minimumFractionDigits: 0,
+  //   maximumFractionDigits: 2,
+  // });
+
+  /*
+  Temporary manual handling of decimal number representation. Will use comma/dot
+  based on language, and a maximal of two decimals. Will not round the decimals.
+  */
+  const formatNumberRegexp = /(\d+)\.(\d{0,2})\d*/g;
+  const decimalMark = language === Language.Norwegian ? ',' : '.';
+  const formatNumberReplacement = `$1${decimalMark}$2`;
+  const vatAmountString = vatAmount
+    .toString()
+    .replace(formatNumberRegexp, formatNumberReplacement);
+  const vatPercentString = vatPercent
+    .toString()
+    .replace(formatNumberRegexp, formatNumberReplacement);
```

Comment:
I understand this is temporary, but still seems like a good idea to refactor into a utility function which lives in `src/utils`? 0.65 is probably several months away - and our ability to upgrade to it even further away.